ocaml/idl/datamodel_lifecycle.ml

```diff
@@ -6,13 +6,13 @@ let prototyped_of_field = function
   | "Repository", "gpgkey_path" ->
       Some "22.12.0"
   | "host", "last_software_update" ->
-      Some "22.19.0-next"
```

Comment:
Is this expected to be part of this PR? It seems unrelated.

Reply:
We just do these updates along the way as the changes are normally automatic.